src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -26,6 +26,7 @@
     public static final Prefix PREFIX_TIME = new Prefix("t/");
 
     // Prefix definitions for property find
+    public static final Prefix PREFIX_PROPERTY_TAG = new Prefix("t/");
```

Comment:
Can reuse the `PREFIX_TYPE` defined earlier in the file instead of defining a duplicate one